neo/rawio/plexon2rawio/plexon2rawio.py

```diff
@@ -103,6 +103,8 @@ def __init__(self, filename, pl2_dll_file_path=None, reading_attempts=25):
 
         # note that this filename is used in self._source_name
         self.filename = pathlib.Path(filename)
+        if (self.filename.is_symlink()):
```

Comment:
```suggestion
        if self.filename.is_symlink():
```
I think this is an okay change regardless so just one code style tweak. Python typically won't do the () around boolean expressions :)